src/core/QCheck.ml

```diff
@@ -775,6 +750,11 @@ module Shrink = struct
     | None -> ()
     | Some shrink -> list_elems shrink l yield
 
+  let string (*?(shrink=char)*) s yield =
+    list ~shrink:char
+      (List.of_seq (String.to_seq s))
+      (fun cs -> yield (String.of_seq (List.to_seq cs)))
```

Comment:
I don't know what's the fastest way to transform a string to a list, did you made any experimentation?

Reply:
I've not done any experimentation.
You are right: this creates a needless intermediate `Seq` representation.

Reply:
This could potentially also make a difference for, e.g., some of the function tests over strings which try many shrinking steps (but not as many as it used to...)
We could also choose to play the "good enough" card :smiley:

Reply:
OK. Now I measured a bit with a conversion micro benchmark.
48d827d includes an update that avoids the intermediate Seq representation.
It doesn't yield any significant speed-up for the current benchmark (but certainly isn't worse)